docs/platforms/javascript/guides/nestjs/features/logging.mdx

```diff
@@ -0,0 +1,27 @@
+---
+title: NestJS Logging
+description: "Learn about integrating Sentry with NestJS logging."
+---
+
+To integrate Sentry logs in your NestJS application, follow the general [logs setup guide](/platforms/javascript/guides/nestjs/logs/).
+
+## Using the Default NestJS Logger
+
+The default NestJS logger uses `console.log` under the hood. If you're using the default logger, you can use the [`consoleIntegration`](/platforms/javascript/guides/nestjs/configuration/integrations/console/) to capture console logs.
+
+Note that this only works if you set the `forceConsole` option in the logger:
+
+```typescript
+const app = await NestFactory.create(AppModule, {
+  logger: new ConsoleLogger({
+    forceConsole: true,
+  })
+});
+```
+
+<Alert level="info">
+
+Further custom support to integrate the NestJS logger with structured attributes may be added in the future.
```

Comment:
I'd link to a GH issue if we're going to add a note like this. Otherwise we should just remove it.